internal/btf/btf_test.go

```diff
@@ -34,13 +34,61 @@ func parseVmLinux(tb testing.TB) (*Spec, error) {
 	return spec, nil
 }
 
-func TestFindType(t *testing.T) {
-	spec, err := parseVmLinux(t)
+func parseELFBTF(tb testing.TB, file string) (*Spec, error) {
+	fh, err := os.Open(file)
+	if err != nil {
+		tb.Fatal(err)
+	}
+	defer fh.Close()
+
+	elf, err := internal.NewSafeELFFile(fh)
+	if err != nil {
+		tb.Fatal(err)
+	}
+
+	sec := elf.Section(".BTF")
+	if sec == nil {
+		tb.Fatalf("ELF %s does not contain .BTF section", file)
+	}
+	rd := sec.Open()
+
+	return loadRawSpec(rd, elf.ByteOrder, nil, nil)
+}
+
+func TestAnyTypesByName(t *testing.T) {
+	testutils.Files(t, testutils.Glob(t, "testdata/relocs-*.elf"), func(t *testing.T, file string) {
+		spec, err := parseELFBTF(t, file)
```

Comment:
Why not just plain `LoadSpecFromReader`?

Reply:
Nice catch, also plumbed the helper into other similar tests.